draft-ietf-core-echo-request-tag.md

```diff
@@ -399,9 +399,9 @@ A single active Echo value with 64 (pseudo-)random bits gives the same theoretic
 
 The security provided by the Echo and Request-Tag options depends on the security protocol used. CoAP and HTTP proxies require (D)TLS to be terminated at the proxies. The proxies are therefore able to manipulate, inject, delete, or reorder options or packets. The security claims in such architectures only hold under the assumption that all intermediaries are fully trusted and have not been compromised.
 
-Servers SHOULD use a monotonic clock to generate timestamps and compute round-trip times. Use of non-monotonic clocks is not secure as the server will accept expired Echo option values if the clock is moved backward. The server will also reject fresh Echo option values if the clock is moved forward. Non-monotonic clocks MAY be used as long as they have deviations that are acceptable given the freshness requirements. If the deviations from a monotonic clock are known, it may be possible to adjust the threshold accordingly.
+Servers SHOULD use a monotonic clock to generate timestamps and compute round-trip times, i.e. a clock source that only increments and never jumps. Use of non-monotonic clocks is not secure as the server will accept expired Echo option values if the clock is moved backward. The server will also reject fresh Echo option values if the clock is moved forward. Non-monotonic clocks MAY be used as long as they have deviations that are acceptable given the freshness requirements. If the deviations from a monotonic clock are known, it may be possible to adjust the threshold accordingly.
 
-Servers SHOULD NOT use wall clock time for timestamps, as wall clock time have large deviations from a monotonic clock. Furthermore, an attacker may be able to affect the server's wall clock time in various ways such as setting up a fake NTP server or broadcasting false time signals to radio-controlled clocks.
+An attacker may be able to affect the server's time in various ways such as setting up a fake NTP server or broadcasting false time signals to radio-controlled clocks.
```

Comment:
I don't think this work. Which time can the attacker change? Not the monotonic clock that we talked about earlier. 

I think we need to talk about wall clock time and system time..... 

Göran: The definitions we looked at yesterday was about "periods of time". Many of the uses in this document is about "points in time"

Reply:
I replace with "server's system time". We can make further changes in the merged version